lib/class-wp-theme-json-resolver.php

```diff
@@ -207,8 +207,8 @@ public static function get_core_data() {
 		}
 
 		$all_blocks = WP_Theme_JSON::ALL_BLOCKS_NAME;
-		$config     = self::get_from_file( __DIR__ . '/experimental-default-theme.json' );
-		self::translate( $config );
+		$config     = self::read_json_file( __DIR__ . '/experimental-default-theme.json' );
+		self::translate_presets( $config );
```

Comment:
Same here, getting PHP fatal errors because of the calls to `translate_presets()` which no longer exists

Reply:
Thanks!

Too many rebases :sweat_smile: